components/access-management/kfam/bindings_test.go

```diff
@@ -44,27 +31,33 @@ func TestGetBindingName(t *testing.T) {
 	if err != nil {
 		log.Fatal(err)
 	}
-
 	*/
-	//format := "--- %s: %s (%s)\n"
-	for _, tt := range tests {
 
-		s, error := getBindingName(tt.in)
+	//Table driven tests
+	var tests = []struct {
+		in       *Binding
+		out      string
+		hasError bool
+	}{
+		{getBindingObject("lalith.vaka@zq.msds.kp.org"), "user-lalith-vaka-zq-msds-kp-org-clusterrole-edit", false},
+		{getBindingObject("397401@zq.msds.kp.org"), "user-397401-zq-msds-kp-org-clusterrole-edit", false},
+		{getBindingObject("lalith.397401@zq.msds.kp.org"), "user-lalith-397401-zq-msds-kp-org-clusterrole-edit", false},
+		{getBindingObject("397401.vaka@zq.msds.kp.org"), "user-397401-vaka-zq-msds-kp-org-clusterrole-edit", false},
+		{getBindingObject("i397401@zq.msds.kp.org"), "user-i397401-zq-msds-kp-org-clusterrole-edit", false},
+	}
 
+	//format := "--- %s: %s (%s)\n"
+	for _, tt := range tests {
+		s, errorReturned := getBindingName(tt.in)
 		if tt.hasError {
 			// expected an error
-			if error == nil {
+			if errorReturned == nil {
 				t.Errorf("got %q, want %q", tt.in, s)
```

Comment:
Last nit:
In this case an unexpected error happened, so the return values shouldn't matter much, but you probably want to see the error.
How about logging: `t.Errorf("Unexpected error occured: %v", errorReturned)`